dll/win32/msgina/lang/pt-PT.rc

```diff
@@ -88,19 +88,19 @@ END
 
 IDD_CHANGEPWD DIALOGEX 0, 0, 275, 166
 STYLE NOT WS_VISIBLE | DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_BORDER | WS_CAPTION | WS_DLGFRAME | WS_POPUP
-CAPTION "Alterar Password"
+CAPTION "Alterar palavra-passe"
 FONT 8, "MS Shell Dlg", 400, 0, 1
 BEGIN
     CONTROL IDI_ROSLOGO, IDC_ROSLOGO, "Static", SS_BITMAP, 0, 0, 275, 54
     LTEXT "Nome de utilizador:", IDC_STATIC, 7, 61, 78, 8
     EDITTEXT IDC_CHANGEPWD_USERNAME, 90, 59, 127, 12, ES_AUTOHSCROLL
     LTEXT "Início de sessão para:", IDC_STATIC, 7, 78, 78, 8
     COMBOBOX IDC_CHANGEPWD_DOMAIN, 90, 75, 127, 144, CBS_DROPDOWNLIST | CBS_SORT | WS_TABSTOP
-    LTEXT "Password antiga:", IDC_STATIC, 7, 95, 78, 8
+    LTEXT "Palavra-passe antiga:", IDC_STATIC, 7, 95, 78, 8
     EDITTEXT IDC_CHANGEPWD_OLDPWD, 90, 92, 127, 12, ES_AUTOHSCROLL | ES_PASSWORD
-    LTEXT "Nova Password:", IDC_STATIC, 7, 111, 78, 8
+    LTEXT "Nova palavra-passe:", IDC_STATIC, 7, 111, 78, 8
     EDITTEXT IDC_CHANGEPWD_NEWPWD1, 90, 109, 127, 12, ES_AUTOHSCROLL | ES_PASSWORD
-    LTEXT "Confirme a nova Password:", IDC_STATIC, 7, 127, 78, 8
+    LTEXT "Confirme a nova palavra-passe:", IDC_STATIC, 7, 127, 78, 8
```

Comment:
I do suggest to prove with a screenshot from reshacker, that this string will fit the 78 width. My gutfeeling is, that it won't.

Reply:
Good shot. Thanks.